drivers/SmartThings/zigbee-dimmer-remote/src/init.lua

```diff
@@ -31,17 +21,16 @@ local function device_init(driver, device)
 end
 
 local zigbee_dimmer_remote_driver_template = {
-    supported_capabilities = {
-        capabilities.battery,
-        capabilities.button,
-        capabilities.switch,
-        capabilities.switchLevel
-    },
-    lifecycle_handlers = {
-      init = device_init,
-    },
-    sub_drivers = { require("zigbee-accessory-dimmer"), require("zigbee-battery-accessory-dimmer")},
-    health_check = false,
```

Comment:
This should remain! Please check the other PRs for this, looks like it was a problem with the script if this line was right next to the health check line.